src/Installer/DockerInstaller.php

```diff
@@ -24,6 +22,6 @@ public function __construct(ChainBuilder $tasks)
      */
     public function install()
     {
-        $this->tasks->getRunner()->run();
+        $this->tasks->getProvider()->getTasks()->getRunner()->run();
```

Comment:
You're talking to the friend of your friend that know a friend that have a friend... It might be simplified right ?